packages/extensions/src/additive-colormap-3D-extensions/additive-blend-extension.js

```diff
@@ -0,0 +1,41 @@
+import BaseExtension from './base-extension';
+
+const _BEFORE_RENDER = '';
```

Comment:
Small nit. Having a shared objects in a different part of the repo with the  combined rendering implementations didn't make sense to me. Keep implementation in the corresponding extension.